#VRML_SIM R2023a utf8

EXTERNPROTO "https://raw.githubusercontent.com/cyberbotics/webots/R2023a/projects/objects/backgrounds/protos/TexturedBackground.proto"
EXTERNPROTO "https://raw.githubusercontent.com/cyberbotics/webots/R2023a/projects/objects/backgrounds/protos/TexturedBackgroundLight.proto"
EXTERNPROTO "https://raw.githubusercontent.com/cyberbotics/webots/R2023a/projects/objects/floors/protos/RectangleArena.proto"
EXTERNPROTO "https://raw.githubusercontent.com/cyberbotics/webots/R2023a/projects/robots/robotis/turtlebot/protos/TurtleBot3Burger.proto"
EXTERNPROTO "https://raw.githubusercontent.com/cyberbotics/webots/R2023a/projects/appearances/protos/BrushedAluminium.proto"
EXTERNPROTO "https://raw.githubusercontent.com/cyberbotics/webots/R2023a/projects/appearances/protos/Parquetry.proto"
EXTERNPROTO "https://raw.githubusercontent.com/cyberbotics/webots/R2023a/projects/objects/traffic/protos/StopSign.proto"
EXTERNPROTO "https://raw.githubusercontent.com/cyberbotics/webots/R2023a/projects/objects/traffic/protos/StopPanel.proto"
EXTERNPROTO "https://raw.githubusercontent.com/cyberbotics/webots/R2023a/projects/objects/apartment_structure/protos/Wall.proto"
EXTERNPROTO "https://raw.githubusercontent.com/cyberbotics/webots/R2023a/projects/objects/factory/containers/protos/PlasticCrate.proto"
EXTERNPROTO "https://raw.githubusercontent.com/cyberbotics/webots/R2023a/projects/objects/factory/containers/protos/WoodenBox.proto"
EXTERNPROTO "https://raw.githubusercontent.com/cyberbotics/webots/R2023a/projects/objects/factory/containers/protos/MetalStorageBox.proto"

WorldInfo {
}
Viewpoint {
  orientation 0.23169145639758654 -0.43610026603664387 -0.8695605942055641 5.186271739488321
  position -2.7477906248965223 3.7458138947631765 0.9303108338249735
}
TexturedBackground {
  texture "empty_office"
  skybox FALSE
  skyColor [
    0.1 0.1 0.1
  ]
}
TexturedBackgroundLight {
  texture "empty_office"
}
RectangleArena {
  translation 0 2.36 0
  floorSize 7 7
  floorAppearance Parquetry {
    type "light strip"
  }
  wallThickness 0.02
  wallHeight 0.6
  wallAppearance BrushedAluminium {
    colorOverride 0 0 0
    IBLStrength 21
  }
}
MetalStorageBox {
  translation -0.26 0.85 0
}
WoodenBox {
  translation 1.20254 1.43826 0.3
  rotation 0 0 1 0.5235996938995747
  name "wooden box(1)"
  size 1.7 0.5 0.6
}
Solid {
  translation 2.93 3.02 0.179973
  rotation 0 1 0 4.692820414042842e-06
  children [
    Shape {
      appearance PBRAppearance {
        baseColor 0 1 0
        metalness 0
      }
      geometry Cylinder {
        height 0.25
        radius 0.09
      }
    }
  ]
  name "green"
}
Solid {
  translation 2.99 -0.12 0.179972
  rotation 0 1 0 4.692820414042842e-06
  children [
    Shape {
      appearance PBRAppearance {
        baseColor 0 1 0
        metalness 0
      }
      geometry Cylinder {
        height 0.25
        radius 0.09
      }
    }
  ]
  name "green(1)"
}
Solid {
  translation -2.33 5.04 0.11
  children [
    Shape {
      appearance PBRAppearance {
        baseColor 0 0 1
        metalness 0
      }
      geometry Cylinder {
        height 0.25
        radius 0.09
      }
    }
  ]
  name "blue"
}
Solid {
  translation -2.27 2.53 0.13
  children [
    Shape {
      appearance PBRAppearance {
        baseColor 0 0 1
        metalness 0
      }
      geometry Cylinder {
        height 0.25
        radius 0.09
      }
    }
  ]
  name "blue(1)"
}
TurtleBot3Burger {
  hidden position_0_0 5680.88334496513
  hidden position_0_1 5239.120096417218
  hidden position_0_2 11.161413842785631
  hidden position2_0_2 112.23589369485711
  hidden position3_0_2 438.355122008419
  hidden linearVelocity_0 0.04378895315603647 -0.21571082005696243 3.608373549034225e-08
  hidden angularVelocity_0 1.3931266625736387e-06 8.01550327244963e-07 -3.5206605040043808e-06
  hidden rotation_5 0 -1 0 5.791944588776189
  hidden linearVelocity_5 0.04379388519808877 -0.21570938862634015 2.426955687819371e-08
  hidden angularVelocity_5 6.536681441900543 1.3269197746343222 -2.2332891571473373e-05
  hidden rotation_6 0.5191804288864019 5.519632768256711e-07 0.8546646607067752 3.141597174346807
  hidden linearVelocity_6 0.04378394096927915 -0.21571135887889606 -5.784136751911905e-08
  hidden angularVelocity_6 6.536674542535759 1.3269537461732996 -2.561073054798725e-05
  hidden rotation_7 -0.8637681589690508 0.3618589415045827 -0.35066033993652523 4.233072887592448
  hidden linearVelocity_7 0.043788987731874444 -0.2157113733930816 1.0510444890506235e-07
  hidden angularVelocity_7 53.92784334827284 10.947246932967639 5.828112288266195e-06
  translation -1.9008401194762325 4.68626518704636 -0.00020956200964894478
  rotation 0.0022046068028676283 0.0026970897431303673 0.9999939326894749 -1.3705251718296827
  controller "myc1"
  extensionSlot [
    TouchSensor {
      translation 0.0996582 0 -0.00522079
      rotation 0 1 0 -1.4398996938995747
      name "front touch sensor"
      boundingObject Cylinder {
        height 0.1
        radius 0.01
      }
    }
    TouchSensor {
      translation -0.004086 0.089544 -0.0110212
      rotation 1 0 0 1.5708003061004252
      name "left touch sensor"
      boundingObject Cylinder {
        height 0.1
        radius 0.01
      }
    }
    TouchSensor {
      translation 0.0096582 -0.07 -0.005221
      rotation 1 0 0 1.5707996938995747
      name "right touch sensor"
      boundingObject Cylinder {
        height 0.1
        radius 0.01
      }
    }
    Camera {
      translation 0.06 0.01 0.01
      recognitionColors [
        0 0 0
      ]
      fieldOfView 1.5
      width 640
      height 640
      recognition Recognition {
        frameThickness 3
      }
    }
  ]
}
StopSign {
  translation -3.41115 0.027381 -0.14
  rotation 0 0 1 1.5708003061004252
  height 0.5
  signBoards [
    StopPanel {
      translation -0.0179959 -0.00168285 -1.35
      rotation 0 0 1 -1.4398996938995747
    }
  ]
}
Wall {
  translation 1.90999 4.27 0
  rotation 0 0 1 1.5707996938995747
  name "wall2"
  size 0.01 3 0.6
}
WoodenBox {
  translation -2.18 1.56 0.3
  name "wooden box(2)"
  size 2.3 0.6 0.6
}
PlasticCrate {
  translation -1.04 3.99 0
  size 1.1 1.1 0.6
  color 0 0 0
}
